src/wp-includes/class-wp-object-cache.php

```diff
@@ -142,6 +142,34 @@ protected function _exists( $key, $group ) {
 		return isset( $this->cache[ $group ] ) && ( isset( $this->cache[ $group ][ $key ] ) || array_key_exists( $key, $this->cache[ $group ] ) );
 	}
 
+	/**
+	 * Serves as a utility function to determine whether a key is valid.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param int|string $key   Cache key to check for validity.
+	 * @return bool Whether the key is valid.
+	 */
+	protected function _valid_key( $key ) {
```

Comment:
A function name like `validate_key()` or `is_valid_key()` would make more sense here.

Reply:
`is_key_valid()` for WP standards I guess.

I think that came about bc someone idiotically added `_exists()` in the last version

Reply:
Renamed to `is_valid_key()` in d896c043d8054bca7826148301608124a8895f6a.